tests/model/test_model.py

```diff
@@ -3634,6 +3634,149 @@ def test__handle_realm_user_event(
         )
 
+    @pytest.mark.parametrize(
+        "person, profile_field_id",
+        [
+            (
+                {
+                    "custom_profile_field": {
+                        "id": 1,
+                        "value": "7237032732",
+                        "rendered_value": "<p>7237032732</p>",
+                    }
+                },
+                "1",
+            ),
+            (
+                {
+                    "custom_profile_field": {
+                        "id": 2,
+                        "value": "Complexity",
+                        "rendered_value": "<p>Complexity</p>",
+                    }
+                },
+                "2",
+            ),
+            (
+                {
+                    "custom_profile_field": {
+                        "id": 3,
+                        "value": "pizza",
+                        "rendered_value": "<p>pizza</p>",
+                    }
+                },
+                "3",
+            ),
+            (
+                {
+                    "custom_profile_field": {
+                        "id": 4,
+                        "value": "0",
+                    }
+                },
+                "4",
+            ),
+            (
+                {
+                    "custom_profile_field": {
+                        "id": 5,
+                        "value": "2023-04-22",
+                    }
+                },
+                "5",
+            ),
+            (
+                {
+                    "custom_profile_field": {
+                        "id": 6,
+                        "value": "https://www.google.com",
+                    }
+                },
+                "6",
+            ),
+            (
+                {
+                    "custom_profile_field": {
+                        "id": 7,
+                        "value": "[11]",
+                    }
+                },
+                "7",
+            ),
+            (
+                {
+                    "custom_profile_field": {
+                        "id": 8,
+                        "value": "gitmaster",
+                    }
+                },
+                "8",
+            ),
+            (
+                {
+                    "custom_profile_field": {
+                        "id": 9,
+                        "value": "he/him",
+                    }
+                },
+                "9",
+            ),
+        ],
+        ids=[
+            "Short Text 1",
+            "Long Text",
+            "Short Text 2",
+            "List of Options",
+            "Date Picker",
+            "Link",
+            "Person Picker",
+            "External Account",
+            "Pronouns",
+        ],
+    )
+    def test__handle_realm_user_event_custom_profile_data(
+        self, person, profile_field_id, model, initial_data
+    ):
+        # id 12 matches initial_data["realm_users"][2] in the initial_data fixture
```

Comment:
Normally the linter complains about capitalized names in functions, since that is reserved for at module-level, but we may have it turned off (we can change it to a case-by-case basis).

In other tests we've erred towards:
```python3
def test_...(a, b, c, ..., constant=5, another=6):
```
or simply defining them in the body of the function, without capitalization.

Reply:
The general changes are an improvement, this is really a nit-pick :)